src/main/webapp/mentors.js

```diff
@@ -55,83 +110,180 @@ function createMentorElement(mentor) {
   mentorCard.className = 'card-holder card h-100 border-primary mb-3';
 
   const mentorCardBody = document.createElement('div');
-  mentorCardBody.className = 'card-body pb-0';
+  mentorCardBody.className = 'card-body pb-0 text-center d-flex flex-column';
 
-  const mentorName = document.createElement('h5');
-  mentorName.className = 'card-title text-primary';
-  mentorName.innerHTML = mentor.name;
-  mentorCardBody.appendChild(mentorName);
+  addImg(mentor.image, mentorCardBody);
```

Comment:
Minor comment: in code like this, I would generally prefer to write something like `mentorCardBody.appendChild(createImg(mentor.image))`. The reasoning is this makes it more clear what modifications we're making to `mentorCardBody`. It also would make the `createImg` etc. functions a bit simpler/easier to test.

Reply:
Sounds good! Thank you. I will just check for null in the parent funciton.

Reply:
Oh yes, that's a good point. The null check would have to be here, or just have the helper functions return empty elements if given null :)